EIPS/eip-969.md

```diff
@@ -0,0 +1,202 @@
+
+Risks include market dominance by a single manufacturer that may utilize production 
+stock to mine themselves, introduce backdoors in their hardware, or otherwise 
+facilitate a 51% attack that would be infeasible. 
+
+This trend towards centralization has a negative effect on network security,
+putting significant control of the network in the hands of only a few entities.
+
+Ethash remains ASIC resistant, however ASIC manufacturer technology is advancing
+and ethash may require further changes in order to remain resistant to unforeseen
+design techniques. This EIP seeks explicitly to buy time during which newly developed
+ASIC technology will face a barrier while more long term mechanisms to ensure 
+continued ASIC resistance can be explored.  
+
+## Specification
+
+If `block.number >= ASIC_MITIGATION_FORK_BLKNUM`, require that the ethash solution 
+sealing the block has been mined using `ethashV2`.
+
+## EthashV2
+
+`ethashV2` will be identical in specification to the current `ethash`(v1) algorithm, with
+the exception of the implementation of `fnv`. 
+
+The new algorithm replaces the 5 current uses of `fnv` inside `hashimoto` with 5 
+separate instances defined as `fnvA`, `fnvB`, `fnvC`, `fnvD`, and `fnvE`, utilizing 
+
+`FNV_PRIME_A=0x10001a7`  
+`FNV_PRIME_B=0x10001ab`  
+`FNV_PRIME_C=0x10001cf`  
+`FNV_PRIME_D=0x10001e3`  
+`FNV_PRIME_E=0x10001f9`  
+
+
+`fnvA` replaces `fnv` in the DAG item selection step
+`fnvB` replaces `fnv` in the DAG item mix step
+`fnvC(fnvD(fnvE` replaces `fnv(fnv(fnv(` in the compress mix step
+
+`fnv` as utilized in DAG-item creation should remain unchanged.
+
+## Agent Changes (Optional Variant)
+ 
+The JSON-RPC `eth_GetWork` call may optionally return the proposed blocks algorithm.
+While a miner or pool may infer the requirement for ethashV2 based on the computed 
+epoch of the provided seedHash, it is beneficial to explicitly provide this
+field so a miner does not require special configuration when mining on a chain
+that chooses not to implement the ASIC_Mitigation hardfork.
+
+Due to compatibility concerns with implementations that already add additional 
+parameters to GetWork, it is desired to add BlockNumber as an explicit 4th parameter 
+as suggested in https://github.com/ethereum/go-ethereum/issues/2333. Algorithm 
+should then be returned as either `"ethash"` or `"ethashV2"` according to the 
+`block.number >= ASIC_MITIGATION_FORK_BLKNUM` criteria. 
+  
+## Rationale
+
+This EIP is aimed at breaking existing ASIC based miners via small changes to the
+existing ethash algorithm.  We hope to accomplish the following:
+
+1. Break existing ASIC based miners.
+2. Demonstrate a willingness to fork in the event of future ASIC miner production.
+
+Goal #1 is something that we can only do probabilistically without detailed
+knowledge of existing ASIC miner design. The known released miner is available for 
+purchase at https://shop.bitmain.com/product/detail?pid=00020180403174908564M8dMJKtz06B7
```

Comment:
Looking at http://eips.ethereum.org/EIPS/eip-969 it seems that the URL is not automatically rendered as a clickable URL.  So may you please change this to `purchase [here](https://shop.bitmain.com/product/detail?pid=00020180403174908564M8dMJKtz06B7)`.

Reply:
I don't think this needs to be a clickable URL, and I think it's actually better if it isn't.

Reply:
Haha, for this Bitmain shop link one, fair enough.